docs/index.rst

```diff
@@ -0,0 +1,28 @@
+Motis utl module documentation
+==============================
+
+MOTIS is an open-source software platform for efficient planning and routing in multi-modal transportation systems.
+GitHub main repository: https://github.com/motis-project/motis
+
+This is the documentation for the `utl` (utility) module.
+
+..
+   Table of contents
+   -----------------
+
+   .. toctree::
+      :maxdepth: 2
+      :caption: Contents:
+
+Logging
+-------
+
+Logs can be produced using the `log()` `struct`::
+
+  utl::log() << "[" << utl::log::str["info"] << "]"        \
+             << "[" << utl::time() << "]"                  \
+             << "[" << FILE_NAME << ":" << __LINE__ << "]" \
+             << " Some message"
```

Comment:
```suggestion
  uLOG(info) << "Some message"; 
```

I hope this would be enough. However, I think the logging style in nigiri is better: it has a tag (for example `nigiri.loader.gtfs`) and uses `libfmt`.

So I think the logging implementation in nigiri would be a better starting point. Since the logging in utl was not used anywhere as far as I remember, it can just be replaced.